### PR TITLE
Add link to Swagger UI to navbar

### PR DESCRIPTION
**Pull request by mik-laj**
I started the server using the following commands.
```
AIRFLOW__WEBSERVER__BASE_URL=http://localhost/my-app airflow webserver -d
```
Screenshot 2020-06-04 at 18 49 13

---
Make sure to mark the boxes below before creating PR: [x]

- [X] Description above provides context of the change
- [X] Unit tests coverage for changes (not needed for documentation changes)
- [X] Target Github ISSUE in description if exists
- [X] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [X] Relevant documentation is updated including usage instructions.
- [X] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

**Review by kaxil — APPROVED** [review]

File: airflow/www/app.py [hunk 1]
```diff
@@ -200,6 +200,9 @@ def init_views(appbuilder):
             appbuilder.add_link("GitHub",
                                 href='https://github.com/apache/airflow',
                                 category="Docs")
+            appbuilder.add_link("REST API Reference",
+                                href='/api/v1./api/v1_swagger_ui_index',
```
Comment (kaxil) [inline airflow/www/app.py:204]:
```suggestion
                                href='/api/v1/api/v1_swagger_ui_index',
```

**Review by kaxil — CHANGES_REQUESTED** [review]

File: airflow/www/app.py [hunk 2]
```diff
@@ -200,6 +200,9 @@ def init_views(appbuilder):
...
+                                category="Docs")
```
Comment (kaxil) [inline airflow/www/app.py:205]:
```suggestion
                                category="API Docs")
```

Comment (mik-laj) [reply airflow/www/app.py:205]: This is a tab in the menu. We already have WWW and other resources in this tab, so it's okay for me.

File: airflow/www/app.py (hunk 1 above)
Comment (mik-laj) [reply airflow/www/app.py:204]: It's required. We need to refer to an endpoint that is in another blueprint. To do this, we must use the syntax `{blueprint_name}.{endpoint_name}`. These two names are generated by connexion, so we have no influence that it do not look cool.

File: airflow/www/app.py (hunk 2 above)
Comment (mik-laj) [reply airflow/www/app.py:205]: I added screenshot to the description.

Comment (kaxil) [reply airflow/www/app.py:205]: oh yes

File: airflow/www/app.py (hunk 1 above)
Comment (kaxil) [reply airflow/www/app.py:204]: aah I thought it was a typo

**Review by kaxil — APPROVED** [review]

File: airflow/www/app.py (hunk 1 above)
Comment (mik-laj) [reply airflow/www/app.py:204]: It took me some time to find out these names, but I added a test to detect if something went wrong.

**Pull request merged**